README.md

```diff
@@ -15,23 +15,32 @@ A: In the spirit of the open source ecosystem and towards the goal of delighting
 
 A: Our primary focus is on security and operational stability. As such, we can't provide specific target dates for features.
 
+**Is it possible to know which features will be included in which release?**
+
+A: Yes! We use [Milestones](https://github.com/giantswarm/roadmap/milestones) to group features by releases so our customers can plan ahead.
+
+https://github.com/giantswarm/roadmap/milestones
+
 **Q: What do the roadmap categories mean?**
 
-A: We have 5 columns in order to allow us to be very clear on the stages of the development process:
+A: We have 6 columns in order to allow us to be very clear on the stages of the development process:
 
 - **Under consideration:**
 This lists the features we are thinking about. We have not committed to them yet, since we are still checking with our customers, upstream and the market as a whole.
```

Comment:
```suggestion
This lists the features we are thinking about. We have not committed to them, since we are still checking with our customers, upstream and the market as a whole.
```